src/Matrix.jl

```diff
@@ -1937,7 +1937,7 @@ function combinations(v::AbstractVector{T}, k::Int) where T
 end
 function _combinations_dfs!(ans::Vector{Vector{T}}, comb::Vector{T}, v::AbstractVector{T}, n::Int, k::Int) where T
    k < 1 && (pushfirst!(ans, comb[:]); return)
-   for m in n:-1:1
+   for m in n:-1:k
```

Comment:
Sorry I was being stupid...

Reply:
Should there have been a test catching this?

Reply:
No the result is correct, it's just a lot of redundant search (happens for example for `combinations(100,99)`, which is how I caught it).